database/index.js

```diff
@@ -1,52 +1,115 @@
-const mysql = require('mysql');
-const Promise = require('bluebird');
-const mysqlConfig = require('./config.js');
-
-const connection = mysql.createConnection(mysqlConfig);
-
-const getAllItems = () => {
-  const q = 'SELECT * FROM items';
-  return new Promise ((resolve, reject) => {
-    connection.query(q, (err, results) => {
-      if (err) {
-        reject(err);
-      } else {
-        resolve(results);
-      }
-    });
-  });
-};
+const Sequelize = require('sequelize');
 
-const getSingleItem = (id) => {
-  const q = 'SELECT * FROM items WHERE id=?';
-  const product_id = id;
-  return new Promise((resolve, reject) => {
-    connection.query(q, product_id, (err, results) => {
-      if (err) {
-        reject(err);
-      } else {
-        resolve(results);
-      }
-    });
-  });
-};
+const sequelize = new Sequelize('postgres://localhost:5432/wawa_melon');
 
-const clearTable = () => {
-  const q = 'TRUNCATE TABLE items';
-  return new Promise((resolve, reject) => {
-    connection.query(q, (err, results) => {
-      if (err) {
-        reject(err);
-      } else {
-        resolve(results);
-      }
-    });
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch((err) => {
+    console.error('Unable to connect to the database:', err);
   });
-};
+
+const { Model } = Sequelize;
+
+class Item extends Model {}
+Item.init({
```

Comment:
Looks like you are defining your schema here using Sequelize.  Make sure that you add parameters for VARCHAR.  I think by default VARCHAR is 255, which might not be long enough for description.

Reply:
Good call, I'll bump it up to 500